includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -856,6 +838,55 @@ public function process_payment( $order_id ) {
 		}
 	}
 
+	/**
+	 * Verifies the phone number to be within valid length.
+	 *
+	 * @param string $phone_number Phone number to be validated.
+	 *
+	 * @throws Process_Payment_Exception
+	 */
+	private function verify_phone_number( string $phone_number ) {
```

Comment:
Do we write test for this change in this PR or the next?

Reply:
There are already tests for the public function in WC_Payment_Gateway_WCPay_Process_Payment_Test, so I don't need to introduce any tests for the new private function.

Reply:
I can introduce tests as part of the Payments_Service, later.